api/client/lib.py

```diff
@@ -184,6 +186,11 @@ def get_data(url, headers, params=None, logger=None):
     """
     base_log_record = dict(route=url, params=params)
     retry_count = 0
+
+    # append version info
+    headers['api_client_version'] = __version__
```

Comment:
Need to update the unit test mocks. Like here:

* https://github.com/gro-intelligence/api-client/blob/3a1cc66e67a20812e7d92ef91824865f927bc971/api/client/lib_test.py#L32

Failing on shippable https://app.shippable.com/github/gro-intelligence/api-client/runs/1773/1/console

Reply:
updated